history/src/postgres/postgres_transaction.rs

```diff
@@ -32,6 +32,81 @@ impl From<&TransactionInfo> for PostgresTransaction {
             cu_consumed: value.cu_consumed.map(|x| x as i64),
             recent_blockhash: value.recent_blockhash.clone(),
             message: value.message.clone(),
+            slot: slot as i64,
+        }
+    }
+
+    pub fn create_statement(schema: &String) -> String {
+        format!(
+            "\
+        CREATE TABLE {}.TRANSACTIONS (
```

Comment:
possible performance improvements:

### Cluster index (recommended)
tldr: will physically collocate data based on values (.slot)

> CREATE INDEX idx_tx_slot ON TRANSACTIONS(slot);
CLUSTER TRANSACTIONS using idx_tx_slot
VACUUM FULL TRANSACTIONS
-- https://www.cybertec-postgresql.com/en/cluster-improving-postgresql-performance/


### GIN/Bitmap
will use bitmap index instead of b-tree which is more optimal for large buckets of data (like transactions per slots)

> INDEX idx_tx_slot  ON TRANSACTIONS USING gin (slot);
-- http://hlinnaka.iki.fi/2014/03/28/gin-as-a-substitute-for-bitmap-indexes/